examples/monitoring/v1alpha1/scylladbmonitoring.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: ScyllaDBMonitoring
+metadata:
+  name: test
```

Comment:
nit: maybe `example` or `monitoring`

Reply:
You could make it match the example ScyllaCluster (`examples/generic/cluster.yaml`), so name it `simple-cluster` and set corresponding selectors (`scylla/cluster: simple-cluster`)? Just so that it works out of the box with the example cluster.

Reply:
I embedded the comment into the value, so the request to match is no longer relevant. I've renamed it to `example` as asked